api/src/main/resources/workbench.yaml

```diff
@@ -1342,11 +1366,12 @@ definitions:
         type: string
 
   # Authorities are a tool used to add discrete permissions to users. We currently
-  # have three authorities that are used:
+  # define the following authorities:
```

Comment:
In the interest of not introducing a new throwaway authority, how about adding the "DEVELOPER" authority here and replacing MANAGE_GROUP with that? This would subsume all other authorities as well, if you all agree. Added the proposed change to this PR.

This is also reusing the MANAGE_GROUPS enum for this purpose. Not an issue in prod since I'm the only one with that authority. I will update the PR description as well.

Reply:
👍 sounds great